bun/bun.go

```diff
@@ -497,3 +499,14 @@ func handleInfo(ex *exchange.Exchange) {
 		"hostname": hostname,
 	})
 }
+
+func handleEnv(ex *exchange.Exchange) {
+	ex.ResponseWriter.Header().Set("Content-Type", "application/json")
+	data := make(map[string]interface{})
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		data[pair[0]] = os.Getenv(pair[0])
+	}
+	jsonContent := util.ToJsonMust(data)
+	ex.WriteBytes(jsonContent)
```

Comment:
The `WriteJson` method might be simpler, and we don't have to set the `Content-Type` header as well.

```suggestion
	util.WriteJson(ex.ResponseWriter, data)
```